PrebidMobile/AdUnits/Native/NativeAd.swift

```diff
@@ -296,6 +296,21 @@ import UIKit
                 Log.debug("Could not open click URL: \(clickUrl)")
             }
         }
+        if let clickTrackers = nativeAdMarkup?.link?.clicktrackers {
```

Comment:
Let's process the click trackers in the same way as implemented in Android PR - once the click URL is successfully opened: 

https://github.com/prebid/prebid-mobile-android/pull/652/files#diff-9cf1ccf8aa5d0eb0a2a198c1261d70fc83d465a1ea3f48351def5e72b971148eR420

In our case is when `openURLWithExternalBrowser` returns `true`.